main/main.cpp

```diff
@@ -98,36 +99,54 @@ Replxx::completions_t hook_shared(std::string const& context, int index, std::ve
     }
     // std::cerr << "[" << context << "][" << prefix << "]" << context.size() << ":" << index << "label[" << label << "]" <<
     // std::endl;
-    bool label_files  = strcasecmp(label.c_str(), "files") == 0;
-    bool label_output = strcasecmp(label.c_str(), "output") == 0;
-    bool label_path   = strcasecmp(label.c_str(), "path") == 0;
-    bool label_odir   = strcasecmp(label.c_str(), "odir") == 0;
-    if (label_files || label_output || label_path || label_odir) {
-      std::string path = ".";
-      auto        pos2 = full_filename.find_last_of('/');
-      std::string filename;
-      if (pos2 != std::string::npos) {
+
+    std::string path = ".";
+    auto        pos2 = full_filename.find_last_of('/');
+    std::string filename;
+    if (pos2 != std::string::npos) {
         path     = full_filename.substr(0, pos2);
         filename = full_filename.substr(pos2 + 1);
         prefix_add += path + "/";
         prefix = filename;
-      } else {
+    } else {
         filename = full_filename;
-      }
+    }
+
+    bool label_files  = strcasecmp(label.c_str(), "files") == 0;
+    bool label_output = strcasecmp(label.c_str(), "output") == 0;
+    bool label_path   = strcasecmp(label.c_str(), "path") == 0;
+    bool label_odir   = strcasecmp(label.c_str(), "odir") == 0;
+    bool label_name   = strcasecmp(label.c_str(), "name") == 0;
+      	
+    if (label_name)
```

Comment:
Remember the coding style is 
If ... {
  ...
Not 
If ...
{

Same for else {